acasa.html

```diff
@@ -0,0 +1,90 @@
+    <title>Pro Barber</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Titlu Pro Barber-->
+    <header class="title">Pro Barber</header>
+    <!--Meniu de navigare-->
+    <nav>
+      <a class="nav-link" href="acasa.html"><h1>Acasa</h1></a>
+      <a class="nav-link" href="servicii.html"><h1>Servicii</h1></a>
+      <a class="nav-link" href="galeriefoto.html"><h1>Galerie foto</h1></a>
+      <a class="nav-link" href="contact.html"><h1>Contact</h1></a>
+    </nav>
+    <!--Partea de descriere-->
+    <main>
+      <div class="top">Stilul clasic nu se demodeaza niciodata!</div>
+      <div class="sub">O atmosfera de neuitat!</div>
+      <div class="subb">Te asteptam in locatia noastra</div>
+    </main>
+    <!--Scurta descriere-->
+    <section class="about">
+      <h2>Despre Pro Barber</h2>
+      <p>
+        Echipa noastra ofera confort,servicii de calitate.La noi aveti parte de
+        o experienta unica si aflati ce inseamna un look clasic adaptat
+        vremurilor moderne. Pro Barber este un loc primitor,cu o atmosfera
+        relaxanta,iar principiul dupa care ne ghidam este satisfacerea celor mai
+        exigente cerinte ale clientilor nostri,astfel incat oricine sa fie
+        incantat de noul look. Frizerii nostri cunosc toate tipurile de par si
+        frizurile perfecte fiecarei fizionomii.Nimic nu este imposibil pentru
+        echipa noastra.
+      </p>
+      <p>
+        Consideram ca frizerul dumneavoastra trebuie sa aiba carisma, sa fie
+        sociabil, sa devina prietenul dumneavoastra, omul caruia puteti sa ii
+        impartasiti orice, sa veniti entuziasmati la programare.
+      </p>
+
+      <p>
+        Desi cateodata, fara intentie, se poate intampla sa se decaleze
+        programarile si sa fiti nevoiti sa asteptati pret de cateva minute, va
+        rugam sa ne iertati. Acest lucru se intampla din dorinta noastra de a
+        excela, de a acorda toata atentia clientului, pentru ca acesta sa plece
+        multumit si cu o frizura impecabila.
+      </p>
+      <p>
+        Datorita experientei frizerilor, a atmosferei placute, a atentiei
+        acordate si a profesionalismului de care dam dovada, credem cu tarie ca
+        oferim un plus de valoare conceptului de frizerie si ne dorim sa fim
+        prima optiune a dumneavoastra.
+      </p>
+      <p>
+        Va garantam ca dupa ce ne-ati calcat pragul o data, o sa va alegeti atat
+        cu un frizer, cat si cu un prieten!
+      </p>
+    </section>
+    <!--Partea de subsol-->
+    <footer>
+      <h2>Detalii de contact</h2>
+      <address>
+        +40774041125 <br />
```

Comment:
Dacă tot pui numărul de telefon în clar, poți să îl faci mai accesibil făcându-l ancoră.

```suggestion
        <a href="tel:+40774041125">+40774041125</a> <br />
```